PyInstaller/hooks/hook-clr.py

```diff
@@ -16,11 +16,19 @@
 import ctypes.util
 from PyInstaller.compat import is_win
 
-
+pyruntime = 'Python.Runtime'
 # Python.net is available only for Windows.
 if is_win:
-    library = ctypes.util.find_library('Python.Runtime')
-    # :todo: Should be issue a warning-message, if the libary is not
-    # found?
+    library = ctypes.util.find_library(pyruntime)
+    datas = []
     if library:
         datas = [(library, '')]
+    else:
+    	import site
+    	from os.path import join as pjoin, exists
```

Comment:
- Please but all imports at the top.
- Please do not use this unusual name `pjoin`, simply keep join.
- Please use `PyInstaller.compat.getsitepackages`